spread/client.go

```diff
@@ -22,6 +22,27 @@ import (
 	"syscall"
 )
 
+var match string = `MATCH() {
+    {
+        set +xu
+    } 2> /dev/null
+
+    [ ${#@} -gt 0 ] || { echo "error: missing regexp argument"; return 1; }
+
+    local stdin="$(cat)"
+    # We are not using pipes here, see:
+    #   https://github.com/snapcore/spread/pull/64
```

Comment:
Can we keep this comment in the Go code and out of the actual string?

It'd also be nice to have just the actual fix in place for this PR, so that we can clearly see the fix independent from the change. I realize the logic is a bit cryptic, but that's done in part because this string is sent along with every single small script that is run on the server. It's usually not the focus.